--- vendor/scribu/scb-framework/Forms.php       2015-10-25 21:19:41.000000000 +0100
+++ patches/Forms2.php  2019-02-19 10:17:03.052324839 +0100
@@ -128,7 +128,7 @@
         * @return string
         */
        public static function form_wrap( $content, $nonce = 'update_options' ) {
-               return html( "form method='post' action=''",
+               return html( "form method='post' action='' enctype='multipart/form-data'",
                        $content,
                        wp_nonce_field( $nonce, '_wpnonce', $referer = true, $echo = false )
                );
